classifier_and_summarizer/summarizer/extractor_output_preprocessor.py

```diff
@@ -0,0 +1,219 @@
+'''This module preprocesses the
+output of the extractor to ready it
+for the later stages of the summarizer
+'''
+
+from nltk.tokenize import sent_tokenize
+from sentence_transformers import SentenceTransformer
+
+from classifier_and_summarizer.summarizer.text_summarizer import TextSummarizer
+from classifier_and_summarizer.summarizer.web_entity_detection import \
+    ImageDescriptionRetriever
+from data_models.contents import ContentType
+
+
+class SentenceWithAttributes:
+    ''' Object to represent a summary sentence
+    or title sentence along with attributes such as
+    embedding or font_style
+    '''
+
+    def __init__(self, text, index, font_style, embedding):
```

Comment:
Let the order of parameters be same as the order in which they are initialized.

Reply:
Updated accordingly!